src/pages/en/reference/cli-reference.md

```diff
@@ -82,30 +147,26 @@ astro telemetry clear
 Make sure you add the `astro telemetry disable` command to your CI scripts.
 :::
 
-## Global Flags
+## Common flags
 
-### `--config path`
+### `--config <path>`
 
 Specifies the path to the config file. Defaults to `astro.config.mjs`. Use this if you use a different name for your configuration file or have your config file in another folder.
 
 ```shell
 astro --config config/astro.config.mjs dev
 ```
 
-### `--root path`
+### `--root <path>`
 
-Specifies the path to the project root. If not specified the current working directory is assumed to be the root.
+Specifies the path to the project root. If not specified, the current working directory is assumed to be the root.
 
 The root is used for finding the Astro configuration file.
```

Comment:
We could change the order so that `--root` appears before `--config`, and then we can say that `--config` "specifies the path to the config file _relative to the root_". This line could sound like root is _only_ used for finding the configuration file, and isn't precise about what that means